metricbeat/module/prometheus/remote_write/_meta/docs.asciidoc

```diff
@@ -7,6 +7,18 @@ remote_write:
   - url: "http://localhost:9201/write"
 ------------------------------------------------------------------------------
 
+TIP: It is recommended to carefully choose the https://prometheus.io/docs/practices/remote_write/#parameters[parameters]
+of remote_write configuration so as to tune the queue according to each specific case and system's characteristics. More
+specifically, `max_shards` option sets the max number of parallelism with which Prometheus will try to send metrics to
+the remote storage. The default value of this is `1000`, which means that in case of Prometheus falls behind the
+remote storage the number of shards will be increased up to `1000` to increase thoughput. This can happen in cases
+of network delays for instance where the whole queue faces issues to consume metrics and finally send metrics to Elasticsearch.
+However increasing the number of parallelism, means that we increase the number of parallel connections from Prometheus
+to Metricbeat. This can lead to resource saturation if the system where Metricbeat runs has limit in `fd` (file descriptors)
+which is lower than `max_shards`. This can be resolved either by setting a higher `fd` limit in the system with `ulimit -n 2048
+` or by setting a lower value for `max_shards`.
```

Comment:
I think something weird happened by having this ` char in a new line

Reply:
hmm, this is weird. I will check it, thank you!